design-system/packages/popover/src/Popover.tsx

```diff
@@ -194,6 +196,24 @@ type DialogProps = {
 export const PopoverDialog = forwardRef<HTMLDivElement, DialogProps>(
   ({ isVisible, children, arrow, ...props }, consumerRef) => {
     const { elevation, radii, shadow, colors } = useTheme();
+    const focusTrapRef = useRef<HTMLDivElement | null>(null);
```

Comment:
popper's provided ref is a function, and not a refObject, we create a refObject here for us to attach our focustrap to